loginsightwebhookdemo/bugzilla.py

```diff
@@ -27,13 +27,12 @@
 
 # Route without <ALERTID> are for LI, with are for vROps
 @app.route("/endpoint/bugzilla", methods=['POST'])
-@app.route("/endpoint/bugzilla/<ALERTID>", methods=['PUT'])
 @app.route("/endpoint/bugzilla/<TOKEN>", methods=['POST'])
-@app.route("/endpoint/bugzilla/<TOKEN>/<ALERTID>", methods=['PUT'])
+@app.route("/endpoint/bugzilla/<TOKEN>/<ALERTID>", methods=['POST','PUT'])
 @app.route("/endpoint/bugzilla/<TOKEN>/<PRODUCT>/<COMPONENT>/<VERSION>", methods=['POST'])
-@app.route("/endpoint/bugzilla/<TOKEN>/<PRODUCT>/<COMPONENT>/<VERSION>/<ALERTID>", methods=['PUT'])
-@app.route("/endpoint/bugzilla/<PRODUCT>/<COMPONENT>/<VERSION>", methods=['POST'])
-@app.route("/endpoint/bugzilla/<PRODUCT>/<COMPONENT>/<VERSION>/<ALERTID>", methods=['PUT'])
+@app.route("/endpoint/bugzilla/<TOKEN>/<PRODUCT>/<COMPONENT>/<VERSION>/<ALERTID>", methods=['POST','PUT'])
+@app.route("/endpoint/bugzilla2/<PRODUCT>/<COMPONENT>/<VERSION>", methods=['POST'])
```

Comment:
Why is there a 2 added in these bugzilla URLs? Is it needed to be unambiguous vs other routes? Or is it to force failure in clients using the old endpoint URLs?